packages/components/code-quality/html-validate/scripts/synergy-element-rules.js

```diff
@@ -0,0 +1,437 @@
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynIcon = {
+  embedded: true,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynInput = {
+  flow: true,
+  focusable: true,
+  formAssociated,
+  interactive: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynMenuItem = {
+  focusable: true,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynMenuLabel = {
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynMenu = {
+  focusable: true,
+  sectioning: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynNavItem = {
+  focusable: true,
+  interactive: true,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynOptgroup = {};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynOption = {
+  focusable: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynPrioNav = {
+  focusable: true,
+  sectioning: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynProgressBar = {
+  formAssociated,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynProgressRing = {
+  formAssociated,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynRadioGroup = {
+  flow: true,
+  focusable: true,
+  formAssociated,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynRadio = {
+  focusable: true,
+  formAssociated,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynRangeTick = {
+  flow: true,
+  phrasing: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynRange = {
+  flow: true,
+  focusable: true,
+  formAssociated,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynSelect = {
+  flow: true,
+  focusable: true,
+  formAssociated,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynSideNav = {
+  flow: true,
+  focusable: true,
+  sectioning: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynSpinner = {
+  formAssociated,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynSwitch = {
+  flow: true,
+  focusable: true,
+  formAssociated,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynTabGroup = {
+  focusable: true,
+  sectioning: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynTabPanel = {
+  sectioning: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynTab = {
+  focusable: true,
+};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynTag = {};
+
+/**
+ * @type {import('html-validate').MetaElement}
+ */
+const SynTextarea = {
+  flow: true,
+  focusable: true,
+  formAssociated,
```

Comment:
I guess should be also interactive

```suggestion
  formAssociated,
  interactive: true,
```

Reply:
Fixed in latest